gradle/support/fetchDependencies.gradle

```diff
@@ -54,15 +54,15 @@ ext.AXML_ZIP = 'https://storage.googleapis.com/google-code-archive-downloads/v2/
 ext.HFS_ZIP = 'https://sourceforge.net/projects/catacombae/files/HFSExplorer/0.21/hfsexplorer-0_21-bin.zip'
 ext.YAJSW_ZIP = 'https://sourceforge.net/projects/yajsw/files/yajsw/yajsw-stable-12.12/yajsw-stable-12.12.zip'
 ext.PYDEV_ZIP = 'https://sourceforge.net/projects/pydev/files/pydev/PyDev%206.3.1/PyDev%206.3.1.zip'
-ext.CDT_ZIP = 'http://www.eclipse.org/downloads/download.php?r=1&protocol=https&file=/tools/cdt/releases/8.6/cdt-8.6.0.zip'
+ext.CDT_ZIP = 'http://eclipse.mirror.rafal.ca/tools/cdt/releases/9.10/cdt-9.10.0/cdt-9.10.0.zip'
```

Comment:
http://download.eclipse.org/tools/cdt/releases/9.10/cdt-9.10.0/cdt-9.10.0.zip
no reason to force a mirror

Reply:
I totally agree. Eclipse was auto forwarding me to the mirror when I was trying to access the tools/cdt/ directory. I have fixed it.